test/cmdlineTests/storage_layout_transient_value_types/input.json

```diff
@@ -0,0 +1,14 @@
+{
+	"language": "Solidity",
+	"sources": {
+		"fileA": {"urls": ["storage_layout_transient_value_types/in.sol"]}
+	},
+	"settings": {
+		"outputSelection": {
+			"fileA": {
+				"A": ["storageLayout"],
+				"": ["storageLayout"]
```

Comment:
We don't have a file-level `storageLayout` output, do we? Note that we don't validate Standard JSON outputs (#12153) so you won't get an error for outputs that do not exist.